beakerx/beakerx/kernel_spec.py

```diff
@@ -19,5 +19,6 @@
 class BeakerXKernelSpec(KernelSpec):
     def __init__(self, **kw):
         super(BeakerXKernelSpec, self).__init__(**kw)
-        args = EnvironmentSettings.read_beakerx_env_settings()
-        self.argv[1:1] = args
+        if 'java' in self.argv:
```

Comment:
should only check for argv[0] is java.  another argument could randomly obscurely be "java"

Reply:
done